### ISSUE TITLE
Resets every new open

### ISSUE THREAD

**Issue by B-Kenji** [graceavery/tamagotchiTemp#18]
If I quit out of the program then reopen it, my pet gets reset.

**Comment by remylavergne** [comment]
Yes, I played a lot yesterday. Today I have to restart from the beginning :'(
A persistence will be awesome !
